README.md

```diff
@@ -506,6 +506,7 @@ Copy the appropriate `![Name](link)` element and paste it in your Markdown file
I+CjxwYXRoIGZpbGwtcnVsZT0iZXZlbm9kZCIgY2xpcC1ydWxlPSJldmVub2RkIiBkPSJNMTI2IDAuMzg1NDE0QzkzLjA0NzIgMC4zODU0MTQgNjIuNzQ0NSA2LjI5ODE3IDQwLjMyNTUgMTYuMjU1N0MyOS4xNDI1IDIxLjIxMjQgMTkuNDMwMiAyNy40MDQ2IDEyLjM0MTUgMzQuODQ3QzUuMjM5NSA0Mi4zMDQyIDAuMTI1IDUxLjc3NjMgMC4xMjUgNjIuODk1OFYyNTMuOTEzTDAuNDY5NSAyNTQuMTA0SDAuMTI1QzAuMTI1IDI2NS4yMjQgNS4yMzk1IDI3NC42OTYgMTIuMzQxNSAyODIuMTUzQzE5LjQzMDIgMjg5LjU5NSAyOS4xNDI1IDI5NS43ODggNDAuMzI1NSAzMDAuNzQ0QzYyLjc0NDUgMzEwLjcwMiA5My4wNDcyIDMxNi42MTUgMTI2IDMxNi42MTVDMTU4Ljk1MyAzMTYuNjE1IDE4OS4yNDIgMzEwLjcwMiAyMTEuNjc0IDMwMC43NDRDMjIyLjg1NyAyOTUuNzg4IDIzMi41NyAyODkuNTk1IDIzOS42NTggMjgyLjE1M0MyNDYuNzYgMjc0LjY5NiAyNTEuODc1IDI2NS4yMjQgMjUxLjg3NSAyNTQuMTA0SDI1MS41MzFMMjUxLjg3NSAyNTMuOTEzVjYyLjg5NThDMjUxLjg3NSA1MS43NzYzIDI0Ni43NiA0Mi4zMDQyIDIzOS42NTggMzQuODQ3QzIzMi41NyAyNy40MDQ2IDIyMi44NTcgMjEuMjEyNCAyMTEuNjc0IDE2LjI1NTdDMTg5LjI0MiA2LjI5ODE3IDE1OC45NTMgMC4zODU0MTQgMTI2IDAuMzg1NDE0Wk0yMCA2Mi44OTU4QzIwIDU5LjgwNzEgMjEuMzc4IDU1LjczMjkgMjYuMDAyMyA1MC44NzkxQzMwLjYzOTggNDUuOTk2IDM3LjkxNCA0MS4wODM0IDQ3LjcwNTcgMzYuNzI5N0M2Ny4yNDk1IDI4LjA1MTggOTQuOTI4NyAyMi40NDc5IDEyNiAyMi40NDc5QzE1Ny4wNzEgMjIuNDQ3OSAxODQuNzUgMjguMDY2NSAyMDQuMjk0IDM2LjcyOTdDMjE0LjA5OSA0MS4wODM0IDIyMS4zNDcgNDUuOTk2IDIyNS45OTggNTAuODc5MUMyMzAuNjM1IDU1LjczMjkgMjMyIDU5Ljc5MjQgMjMyIDYyLjg5NThDMjMyIDY1Ljk4NDYgMjMwLjYyMiA3MC4wNTg4IDIyNS45OTggNzQuOTEyNUMyMjEuMzYgNzkuNzk1NyAyMTQuMDg2IDg0LjcwODMgMjA0LjI5NCA4OS4wNjJDMTg0Ljc1IDk3LjczOTkgMTU3LjA3MSAxMDMuMzQ0IDEyNiAxMDMuMzQ0Qzk0LjkyODcgMTAzLjM0NCA2Ny4yNDk1IDk3LjcyNTIgNDcuNzA1NyA4OS4wNjJDMzcuOTAwNyA4NC43MDgzIDMwLjY1MyA3OS43OTU3IDI2LjAwMjMgNzQuOTEyNUMyMS4zNjQ4IDcwLjA1ODggMjAgNjUuOTk5MyAyMCA2Mi44OTU4Wk0yMzIgMTU4LjVWOTcuNzY5M0MyMjUuNjM3IDEwMi41MTIgMjE4LjgyIDEwNi40NTggMjExLjY3NCAxMDkuNTM2QzE4OS4yNDIgMTE5LjQ5MyAxNTguOTUzIDEyNS40MDYgMTI2IDEyNS40MDZDOTMuMDQ3MiAxMjUuNDA2IDYyLjc0NDUgMTE5LjQ5MyA0MC4zMjU1IDEwOS41MzZDMzMuMTgwMiAxMDYuNDU4IDI2LjM2MzUgMTAyLjUxMiAyMCA5Ny43NjkzVjE1OC41QzIwIDE2MS41ODkgMjEuMzc4IDE2NS42NjMgMjYuMDAyMyAxNzAuNTE3QzMwLjYzOTggMTc1LjQgMzcuOTE0IDE4MC4zMTIgNDcuNzA1NyAxODQuNjUxQzY3LjI0OTUgMTkzLjMyOSA5NC45Mjg3IDE5OC45NDggMTI2IDE5OC45NDhDMTU3LjA3MSAxOTguOTQ4IDE4NC43NSAxOTMuMzI5IDIwNC4yOTQgMTg0LjY1MUMyMTQuMDk5IDE4MC4zMTIgMjIxLjM0NyAxNzUuNCAyMjUuOTk4IDE3MC41MzFDMjMwLjYzNSAxNjUuNjc4IDIzMiAxNjEuNjAzIDIzMiAxNTguNVpNMjAgMTkzLjM3M0MyNS44Njk4IDE5Ny44NiAzMi43NTk4IDIwMS43ODcgNDAuMzI1NSAyMDUuMTRDNjIuNzQ0NSAyMTUuMDk4IDkzLjA0NzIgMjIxLjAxIDEyNiAyMjEuMDFDMTU4Ljk1MyAyMjEuMDEgMTg5LjI0MiAyMTUuMDk4IDIxMS42NzQgMjA1LjE0QzIxOC44MiAyMDIuMDYyIDIyNS42MzcgMTk4LjExNiAyMzIgMTkzLjM3M1YyNTMuOTEzTDIzMi4xNzIgMjU0LjEwNEgyMzJDMjMyIDI1Ny4xOTMgMjMwLjYyMiAyNjEuMjY3IDIyNS45OTggMjY2LjEyMUMyMjEuMzYgMjcxLjAwNCAyMTQuMDg2IDI3NS45MTcgMjA0LjI5NCAyODAuMjdDMTg0Ljc1IDI4OC45NDggMTU3LjA3MSAyOTQuNTUyIDEyNiAyOTQuNTUyQzk0LjkyODcgMjk0LjU1MiA2Ny4yNDk1IDI4OC45MzQgNDcuNzA1NyAyODAuMjdDMzcuOTAwNyAyNzUuOTE3IDMwLjY1MyAyNzEuMDA0IDI2LjAwMjMgMjY2LjEyMUMyMS4zNjQ4IDI2MS4yNjcgMjAgMjU3LjIwOCAyMCAyNTQuMTA0SDE5LjgyNzdMMjAgMjUzLjkxM1YxOTMuMzU5VjE5My4zNzNaIiBmaWxsPSJibGFjayIvPgo8L3N2Zz4K)`               |
```

Comment:
Hey @frivajica!
Thanks for taking the time to contribute.
This URL is too long, can we get something shorter?